proposals/xxxx_carvel-package/README.md

```diff
@@ -0,0 +1,231 @@
+   [the docs](https://carvel.dev/kapp/) for more information.
+- `imgpkg` is a Carvel provided CLI tool that provides a mechanism for collecting configuration and 
+   OCI images into a bundle that can be deployed on a cluster.
+- `kapp-controller` is a server side component managing software in the form of Carvel `App`s delivered
+   via Carvel `Package`s. See [the docs](https://carvel.dev/kapp-controller/) for more information.
+- `PackageRepository` is a custom resource that configures `kapp-controller` with a set of 
+  `Package`s on a cluster.  See [the docs](https://carvel.dev/kapp-controller/docs/v0.47.x/packaging/#package-repository)
+   for more information.
+- `Package` is a custom resource that represents configuration in the form of metadata and OCI images
+   that may be used to deliver software onto a Kuberentes cluster. See [the docs](https://carvel.dev/kapp-controller/docs/v0.47.x/packaging/#package)
+   for more information. 
+- `PackageMetadata` is a custom resource describing attributes for a `Package`.  See [the docs](https://carvel.dev/kapp-controller/docs/v0.47.x/packaging/#package-metadata)
+   for more information.
+
+## Proposal
+
+Allow Pinniped to be deployed onto a Kuberentes cluster through the mechanism of two Carvel `Packages`,
+a Supervisor and a Concierge package. These may be delivered via a `PackageRepository` resource and installed
+via `PackageInstall` custom resources, and `Secret`s containing `Package` configuration.
+
+Conceptually, cluster managers would make the Pinniped software available on the 
+cluster by deploying the PackageRepository:
+
+```bash
+# Deploy the Pinniped PackageRepository to the globally available
+# namespace watched by kapp-controller for new Packages 
+kapp deploy --app pinniped-package-repository --file <pinniped-release-files>/pinniped-package-repository.yaml
+```
+
+Then developers responsible for deploying Supervisor and Concierge would create the 
+appropriate resources to successfully deploy the PackageInstall and Packages for both 
+`Supervisor` and `Concierge`:
+
+```bash 
+# create a Service account and RBAC for the PackageInstall 
+vim supervisor-service-and-rbac.yaml
+kapp deploy --app supervisor-rbac --file supervisor-service-and-rbac.yaml
+vim concierge-service-and-rbac.yaml
+kapp deploy --app concierge-rbac --file concierge-service-and-rbac.yaml
+
+# create a PackageInstall and a Secret for configuring the Concierge
+vim supervisor-package-install-bundle.yaml
+kapp deploy --app supervisor --file supervisor-package-install-bundle.yaml
+vim concierge-package-install-bundle.yaml
+kapp deploy --app supervisor --file concierge-package-install-bundle.yaml
+```
+
+The `PackageRepository` will contain a series of versions of each of the Packages for Supervisor 
+and Concierge.
+
+The `PackageInstall` files will contain `constraints` representing acceptable versions of both the 
+`Supervisor` and `Concierge. For example:
+
+```yaml
+spec:  
```

Comment:
Would you mind including `apiVersion` and `kind` on these example resources? Would really help me figure out how the pieces come together.